SWLOR.Game.Server/Feature/QuestDefinition/HutlarQuestDefinition.cs

```diff
@@ -155,5 +156,42 @@ private void StupendousSlugBile()
                 .AddItemReward("slug_shake", 1);
         }
 
+        private void BreakTheByysk()
+        {
+            _builder.Create("break_the_byysk", "Break the Byysk")
+
+                .AddState()
+                .SetStateJournalText("Sharene wants you to kill One Hundred Byysk. Of you go!")
```

Comment:
Bunch of grammar issues here.

"Sharene wants you to kill one hundred Byysk. Off you go!"